pkg/reconciler/openshift/namespace/namespace.go

```diff
@@ -0,0 +1,279 @@
+/*
+Copyright 2023 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package namespace
```

Comment:
Just a thought/observation
I see the goal of this is to restrict SCC on that perticular namespace and the functionality changes is for SCC not to namespace

I think package name namespace is confusing bit about the goal of the folder

can we rename it ?

Reply:
this package has a validating webhook for namespace, and that validating webhook currently validates admission for SCC configs
i think if we keep the scope for the namespace, we have more flexibility to add future configurations for any namespace related functionality instead of limiting to SCC, wdyt?

Reply:
ok got it :+1: